### PR TITLE
Added makaut page through dynamic

### PR DESCRIPTION
**Pull request by SUPRIYODEY886**
Check it.

### CODE REVIEW

File: css/makaut.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
+body {
+
```
Comment (sangy05) [inline css/makaut.css]: Empty Body??

File: practice3.html [hunk 2]
```diff
@@ -22,6 +22,10 @@ <h1> My Website </h1>
 		<a href="#"> Help </a>
 	</div>
 
+	<div class = "nav">
+		<p>Hi Bapon</p>
```
Comment (sangy05) [inline practice3.html]: Can remove this ?

File: makaut.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title> Makaut </title>
```
Comment (sangy05) [inline makaut.html]: Change 'Makeup' to your name.

File: css/makaut.css (hunk 1 above)
Comment (SUPRIYODEY886) [reply css/makaut.css]: Resolved

File: practice3.html (hunk 2 above)
Comment (SUPRIYODEY886) [reply practice3.html]: Done

File: makaut.html (hunk 3 above)
Comment (SUPRIYODEY886) [reply makaut.html]: Done

**Pull request merged**